sips/sip-021/sip-021-nakamoto.md

```diff
@@ -36,6 +36,23 @@ In this proposal, Stacks block production would no longer be tied to miner elect
 
 This proposal, dubbed the "Nakamoto" release, represents a substantial architectural change to the current Stacks blockchain. If adopted, the Stacks major version would be bumped from 2 to 3. The first Nakamoto release would be 3.0.0.0.0.
 
+# Addendum
```

Comment:
I think this addendum section can go at the bottom instead of at the top.

Reply:
The only Addendum I could find was on top of the document. I am fine with both.

Reply:
I recommend moving addendum to the end of ratified SIP.

Reply:
I just pushed a commit moving this to the end.